Gemfile

```diff
@@ -1,45 +1,42 @@
-source 'https://rubygems.org'
+#
+# The default setup of Huginn is pretty large and may not fit on
+# servers with lower RAM.  To conserve RAM, you can turn off some of
+# the Agent types you don't need and avoid loading their dependencies.
+# It is also a good idea to not install gems only for development on
+# production.  To do this, list the gem groups you want to opt out of
+# after `--without` when you run `bundle` on your running environment.
+# e.g.:
+#
+#     bundle install --deployment --without development test basecamp human_task jabber mqtt weibo
```

Comment:
OK, I've rephrased the instructions.  Feel free to suggest improvement to it!
And we'll have to update Wiki pages later anyway.